dtfs-central-api/api-tests/v1/utilisation-reports/fee-record-corrections/put-fee-record-correction.api-test.ts

```diff
@@ -0,0 +1,169 @@
+import { HttpStatusCode } from 'axios';
+import { Response } from 'supertest';
+import {
+  FEE_RECORD_STATUS,
+  FeeRecordCorrectionEntityMockBuilder,
+  FeeRecordCorrectionTransientFormDataEntityMockBuilder,
+  FeeRecordEntityMockBuilder,
+  RECONCILIATION_IN_PROGRESS,
+  ReportPeriod,
+  UtilisationReportEntityMockBuilder,
+} from '@ukef/dtfs2-common';
+import { withSqlIdPathParameterValidationTests } from '@ukef/dtfs2-common/test-cases-backend';
+import { ObjectId } from 'mongodb';
+import { testApi } from '../../../test-api';
+import { SqlDbHelper } from '../../../sql-db-helper';
+import wipeDB from '../../../wipeDB';
+import { aBank } from '../../../../test-helpers';
+import { replaceUrlParameterPlaceholders } from '../../../../test-helpers/replace-url-parameter-placeholders';
+import { mongoDbClient } from '../../../../src/drivers/db-client';
+
+console.error = jest.fn();
+
+interface CustomErrorResponse extends Response {
+  body: { errors: { msg: string }[] };
+}
+
+interface CustomSuccessResponse extends Response {
+  body: {
+    sentToEmails: string[];
+    reportPeriod: ReportPeriod;
+  };
+}
+
+const BASE_URL = '/v1/bank/:bankId/fee-record-corrections/:correctionId';
+
+describe(`PUT ${BASE_URL}`, () => {
+  const bankId = '1';
+  const correctionId = 2;
+  const userId = new ObjectId().toString();
+
+  const reportPeriod = { start: { month: 3, year: 2023 }, end: { month: 4, year: 2023 } };
+
+  const paymentOfficerTeamEmails = ['email1@ukexportfinance.gov.uk'];
+
+  const mockBank = {
+    ...aBank(),
+    id: bankId,
+    _id: new ObjectId(),
```

Comment:
should it have both these properties?

Reply:
I guess we probably don't need to set the second as it is irrelevant to our tests, but generally banks do have both these fields, they have a mongo id as the field '_id' and a separate id that is a string of digits as the 'id' field